infrastructure/kctf/scripts/cluster/start.sh

```diff
@@ -11,10 +11,21 @@ MAX_NODES="2"
 NUM_NODES="1"
 MACHINE_TYPE="n2-standard-4"
 
-gcloud container clusters create --enable-network-policy --enable-autoscaling --min-nodes ${MIN_NODES} --max-nodes ${MAX_NODES} --num-nodes ${NUM_NODES} --create-subnetwork name=kctf-${CLUSTER_NAME}-subnet --no-enable-master-authorized-networks --enable-ip-alias --enable-private-nodes --master-ipv4-cidr 172.16.0.32/28 --enable-autorepair --preemptible --machine-type ${MACHINE_TYPE} ${CLUSTER_NAME}
+EXISTING_CLUSTER=$(gcloud container clusters list --filter "name=${CLUSTER_NAME}" --format 'get(name)')
 
-gcloud compute routers create kctf-${CLUSTER_NAME}-nat-router --network=default --region ${ZONE::-2}
-gcloud compute routers nats create kctf-${CLUSTER_NAME}-nat-config --router-region europe-west4 --router kctf-${CLUSTER_NAME}-nat-router --nat-all-subnet-ip-ranges --auto-allocate-nat-external-ips
+if [ -z "${EXISTING_CLUSTER}" ]; then
```

Comment:
shouldn't the cluster-start script abort if the cluster exists? I.e. is this mainly useful for dev or is it useful for normal usage too?

Reply:
creating and deleting a cluster is the first operation and is very slow, so if you face issues below (like missing permissions, forgot to enable an API, etc), its super slow to fix them by cycling through delete/create.